compiler-plugin/testData/api/mangledNames.kt

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2020 Google LLC
+ * Copyright 2010-2020 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// WITH_RUNTIME
+// TEST PROCESSOR: MangledNamesProcessor
+// EXPECTED:
+// mainPackage.Foo -> declarations
+// get-normalProp -> getNormalProp
+// set-normalProp -> setNormalProp
+// get-inlineProp -> getInlineProp-HRn7Rpw
+// set-inlineProp -> setInlineProp-E03SJzc
+// get-internalProp -> getInternalProp$mainModule
+// set-internalProp -> setInternalProp$mainModule
+// get-internalInlineProp -> getInternalInlineProp-HRn7Rpw$mainModule
+// set-internalInlineProp -> setInternalInlineProp-E03SJzc$mainModule
+// normalFun -> normalFun
+// inlineReceivingFun -> inlineReceivingFun-E03SJzc
+// inlineReturningFun -> inlineReturningFun-HRn7Rpw
+// internalInlineReceivingFun -> internalInlineReceivingFun-E03SJzc$mainModule
+// internalInlineReturningFun -> internalInlineReturningFun-HRn7Rpw$mainModule
+// fileLevelInternalFun -> fileLevelInternalFun
+// fileLevelInlineReceivingFun -> fileLevelInlineReceivingFun-E03SJzc
+// fileLevelInlineReturningFun -> fileLevelInlineReturningFun
```

Comment:
Just curious, why this and `fileLevelInternalInlineReturningFun`  isn't mangled?

Reply:
i honestly don't know 🤦 
I got these outputs from compiling the same input on a sample project and it is not mangled there.

```
➜  yigit-google-ksp git:(mangled-names) javap -p ~/src/ManglingPlayground/mainModule/build/classes/kotlin/main/mainPackage/InputKt.class
Compiled from "input.kt"
public final class mainPackage.InputKt {
  public static final void fileLevelInternalFun();
  public static final void fileLevelInlineReceivingFun-E03SJzc(java.lang.String);
  public static final java.lang.String fileLevelInlineReturningFun();
  public static final void fileLevelInternalInlineReceivingFun-E03SJzc(java.lang.String);
  public static final java.lang.String fileLevelInternalInlineReturningFun();
}
```
vs it is mangled when it is on a class:
```
➜  yigit-google-ksp git:(mangled-names) javap -p ~/src/ManglingPlayground/mainModule/build/classes/kotlin/main/mainPackage/Foo.class    
Compiled from "input.kt"
public final class mainPackage.Foo {
  private java.lang.String normalProp;
  private java.lang.String inlineProp;
  private java.lang.String internalProp;
  private java.lang.String internalInlineProp;
  public final java.lang.String getNormalProp();
  public final void setNormalProp(java.lang.String);
  public final java.lang.String getInlineProp-HRn7Rpw();
  public final void setInlineProp-E03SJzc(java.lang.String);
  public final java.lang.String getInternalProp$mainModule();
  public final void setInternalProp$mainModule(java.lang.String);
  public final java.lang.String getInternalInlineProp-HRn7Rpw$mainModule();
  public final void setInternalInlineProp-E03SJzc$mainModule(java.lang.String);
  public final void normalFun();
  public final void inlineReceivingFun-E03SJzc(java.lang.String);
  public final java.lang.String inlineReturningFun-HRn7Rpw();
  public final void internalInlineReceivingFun-E03SJzc$mainModule(java.lang.String);
  public final java.lang.String internalInlineReturningFun-HRn7Rpw$mainModule();
  public mainPackage.Foo();
}
```